gcalcli

```diff
@@ -1697,10 +1728,17 @@ class gcalcli:
 
         event = {}
         event['summary'] = stringToUnicode(eTitle)
-        event['start'] = {'dateTime': eStart,
-                          'timeZone': self.cals[0]['timeZone']}
-        event['end'] = {'dateTime': eEnd,
-                        'timeZone': self.cals[0]['timeZone']}
+
+        if FLAGS.allday:
```

Comment:
You don't set `dateTime`/`timeZone` to `None` here, so I'm assuming it's not needed above.  If not required I would remove those lines just to reduce length.